inference-engine/src/mkldnn_plugin/nodes/mkldnn_bucketize_node.h

```diff
@@ -15,9 +15,13 @@ class MKLDNNBucketizeNode : public MKLDNNNode {
 
     void getSupportedDescriptors() override {};
     void initSupportedPrimitiveDescriptors() override;
-    void createPrimitive() override {};
+    void createPrimitive() override;
     void execute(mkldnn::stream strm) override;
     bool created() const override;
+    void executeDynamicImpl(mkldnn::stream strm) override {
+        execute(strm);
+    }
+    void prepareParams() override;
```

Comment:
please also override `shapeInfer` since output shape the same as input
https://github.com/openvinotoolkit/openvino/blob/24675fd8ea11f964cd732eda1f27cdaa3d18dd20/inference-engine/src/mkldnn_plugin/nodes/mkldnn_reorder_node.cpp#L399-L401

Reply:
Good idea, Added